internal/artifact/image.go

```diff
@@ -38,7 +38,7 @@ func ImageRun(cliCtx *cli.Context) error {
 
 	// initialize config
 	if err = c.Init(true); err != nil {
-		return xerrors.Errorf("failed to initialize options: %w", err)
+		return nil
```

Comment:
I think we should keep it as is because `Init` may return another error.

Reply:
If I keep it, when I use without parameters, appears after the "help", the error message, any idea how to hide it? 
```
   --help, -h         show help (default: false)
   --version, -v      print the version (default: false)
2020/06/30 21:43:31 failed to initialize options: arguments error
```